sources/scrapy_scraper/README.md

```diff
@@ -0,0 +1,3 @@
+# 🕸️ Scrapy source
+
+
```

Comment:
@sultaniman could you please add a quick "Run the pipeline" section here. For starters, how to run `scrapy_scraper_pipeline.py` with `python` or `scrapy`?

Reply:
Yes I will definitely add more contents here, just have been trying to figure out testing of new sources.

Reply:
Got it, no need for a big writeup: just how to run the example script since this is a special case. (+ prerequisites, if any)